blockchain/network.py

```diff
@@ -518,13 +520,35 @@ def submit_verification(self, block_id, phase, block_record):
 
     def phase_1_message(self, phase_1):
         """ submit phase_1 block for phase_2 validation_phase """
-        self.connection_manager.processing_node.notify(2, phase_1_info=phase_1)
+        phase_1_info = {
+            'record': thrift_record_to_dict(phase_1.record),
+            'verification_info': map(thrift_transaction_to_dict, phase_1.transactions)
+        }
+        self.connection_manager.processing_node.notify(2, phase_1_info=phase_1_info)
 
     def phase_2_message(self, phase_2):
-        self.connection_manager.processing_node.notify(3, phase_2_info=phase_2)
+        phase_2_info = {
+            'record': thrift_record_to_dict(phase_2.record),
+            'verification_info': {
+                'valid_txs': map(thrift_transaction_to_dict, phase_2.valid_txs),
+                'invalid_txs': map(thrift_transaction_to_dict, phase_2.invalid_txs),
+                'business': phase_2.business,
+                'deploy_location': phase_2.deploy_location
+            }
+        }
+        self.connection_manager.processing_node.notify(3, phase_2_info=phase_2_info)
 
     def phase_3_message(self, phase_3):
-        self.connection_manager.processing_node.notify(4, phase_3_info=phase_3)
+        phase_3_info = {
+            'record': thrift_record_to_dict(phase_3.record),
```

Comment:
Thank you for the feedback. I will create constants for these fields and look for anywhere else we should be using constants.